internal/ui/input_glfw.go

```diff
@@ -78,7 +87,18 @@ var glfwMouseButtonToMouseButton = map[glfw.MouseButton]MouseButton{
 func (i *Input) update(window *glfw.Window, scale float64) {
 	i.m.Lock()
 	defer i.m.Unlock()
-
+	if i.runebuffer == nil {
+		i.runebuffer = make([]rune, 0, 1024)
+		window.SetCharModsCallback(func(w *glfw.Window, char rune, mods glfw.ModifierKey) {
+			if unicode.IsPrint(char) {
```

Comment:
What if we don't have `IsPrint`? As `unicode.IsPrint('\n')` is false, '\n' can never come to the rune buffer?